_posts/2018-07-10-introducing-accesability-for-teams.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Introducing Accessibility for Teams"
+date: 2018-07-10
+authors:
+- maya
+tags:
+- accessibility
+- design
+excerpt: "The Accessibility Guild in the Technology Transformation Services (TTS) at the U.S. General Services Administration (GSA) set out to understand how people in different roles practice accessibility. We asked designers, developers, and product managers across our organization to share their accessibility practices, from self-testing to asking for help."
+image: /assets/blog/accessibility/accesability-logo.png
+---
+Accessibility is a crucial part of government product design. First,
+[it's the law](https://www.access-board.gov/the-board/laws/rehabilitation-act-of-1973#508).
+Federal agencies face legal consequences when they don’t meet
+accessibility requirements. Second, it affects us all. Whether you have
+a motor disability, you sprained your wrist playing dodgeball, you need
+a building to have a ramp for your wheelchair or stroller, or you
+literally just have your hands full, we all find ourselves unable to do
+certain things at different points in our lives. Accessible products are
+better products for everyone.
+
+But accessibility is hard: It comes across as a set of complex rules
+that are hard to follow. Not everyone feels confident that they’re doing
+it right. It’s difficult to prioritize alongside other work and project
+needs. How do you make sure you’re building products that are accessible
+and inclusive?
+
+The Accessibility Guild in the Technology Transformation Services (TTS)
+at the U.S. General Services Administration (GSA) set out to understand
+how people in different roles practice accessibility. We asked
+designers, developers, and product managers across our organization to
+share their accessibility practices, from self-testing to asking for
+help. We heard about the barriers that can stand in the way of making
+products more accessible, from lack of knowledge to lack of buy-in.
+
+We distilled what we learned into a [quick-start guide for embedding
+accessibility and inclusive design](https://accessibility.digital.gov/)
+into a team's workflow.
+
+Each person on a team, whether you’re a manager, designer, or developer,
+has a role to play. Your responsibilities are different depending on
+your role. So that’s how we structured the guide, with a separate
+section for each of five roles:
+
+-   Product management
+
```

Comment:
Shouldn't this be without an extra space between list items? Adding the space inserts another paragraph tag `<p>`. For this and the list below.